clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreBufferService.java

```diff
@@ -80,8 +80,10 @@ public StoreBufferService(
     }
     this.isSorted = sorted;
     this.leaderRecordHandler = queueLeaderWrites ? this::queueLeaderRecord : StoreBufferService::processRecord;
+    String metricName = sorted ? "StoreBufferServiceSorted" : "StoreBufferServiceUnsorted";
     this.storeBufferServiceStats = new StoreBufferServiceStats(
         metricsRepository,
+        metricName,
```

Comment:
Nit: maybe changed to `metricNamePrefix`?

Reply:
For sure. Updated.